src/app/components/feed/jh-main-feed.directive.js

```diff
@@ -33,8 +35,44 @@
           Janus.attachMediaStream(video, newVal);
         }
       });
+      
+      $timeout(function(){
+        var video = $('video', element)[0];
+        var mainFeed = document.getElementById('main-feed');
+        video.addEventListener('dblclick', function() {
```

Comment:
@dgdavid All right, I'll do that if you say so. I don't know what I was thinking while doing that. Thanks for review and explanation.